docs/react-v9/contributing/api-extractor.md

```diff
@@ -0,0 +1,25 @@
+### What is API Extractor?
+
+[API Extractor](https://api-extractor.com/) generates an api.md file containing the public API for a package. `@fluentui/react` and related packages as well as `@fluentui/web-components` use API Extractor to ensure that API changes must be reviewed before being merged into master. (`@fluentui/react-northstar` and related packages do not use API Extractor as of writing.)
+
+### Build Tasks
+
+As part of a package's build process, the api-extractor task checks the package's public API with the package's auto-generated .api.md file.
+
+If building locally, the API file will be updated automatically (in `7.0` and `master`). Be sure to commit the updates with your other changes!
+
+In PR builds, the task will fail if it detects missing API file updates.
+
+(Previously it was necessary to run a separate command `yarn update-api` to update the API files, but we decided to streamline this and do the update automatically as part of the build instead.)
+
+To generate the API markdown file, first commit your changes, then run:
+
+`yarn nx run <package-name>:generate-api`
```

Comment:
Let's stick with `build` now because if one want's to update, lets say v8-v9 or v0-v9 migration packages API surface, this will fail because those old libraries have no `generate-api` task. This will hopefully change in the future as `build` does lot of things which shouldn't be part of build.

```suggestion
`yarn nx run <package-name>:build`
```

Reply:
agreed :-)